uint32 frame_id
uint32 id
uint32[] bb_width
uint32[] bb_height
uint32[] bb_x
uint32[] bb_y
